gammapy/maps/wcs.py

```diff
@@ -21,6 +21,28 @@
     'WcsGeom',
 ]
 
+def _check_width(width):
+    """Convert input width as tuple of float to pass to WcsGeom.create"""
+    if isinstance(width, (Angle,Quantity)):
```

Comment:
Angle is a Quantity sub-class. You can just write `isinstance(width, Quantity)` here with same effect.